### PR TITLE
fix RemovedInDjango19Warning warnings

### PR DESCRIPTION
**Pull request by muhammad-ammar**
TNL-3450

All the warnings are gone except one IPAddressField warning and for that we need to create a new migration. As per discussion with Usman, he will update IPAddressField to GenericIPAddressField along with other changes he wants to make.

### CODE REVIEW

**Comment by muhammad-ammar** [comment]
@nedbat @symbolist @macdiesel @muzaffaryousaf @doctoryes. Please have a look. Thanks!

**Comment by muhammad-ammar** [comment]
Oh..there are some `RemovedInDjango19Warning` coming from third party packages like `python-social-auth`. 

What should we do with those warnings?

**Comment by symbolist** [comment]
@muhammad-ammar Are there any updates for the `python-social-auth` package?

**Comment by symbolist** [comment]
:+1:

File: setup.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def build_media_pattern(base_folder, file_extension):
 
 setup(
     name = "django-wiki",
-    version = "0.0.2",
+    version = "0.0.3",
```
Comment (nedbat) [inline setup.py]: This version bumping makes me nervous: the upstream repo is at "0.1.dev"  I guess we've pretty much abandoned any hope of re-joining master some day.

**Comment by nedbat** [comment]
:+1:  We now have three stacked PRs, we should merge them down.

**Pull request merged**